### PR TITLE
William2

### PR DESCRIPTION
**Pull request by wakerman17**
Rasp access and more clean code

### CODE REVIEW

**Comment by wakerman17** [comment]
I myself thinks it's good, but I can't approve, i created the pull request.

File: cloudserver/app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -43,9 +90,65 @@ public function index()
...
 						->first();
 		}
+		
+
 		return 	view('/home')
 				->with('user_devices', $user_devices)
-				->with('ip', $ip);
-        //return view('home');
+				->with('ip', $ip)
+				->with('raspberrys', $raspberrys);*/
```
Comment (MexxII) [inline cloudserver/app/Http/Controllers/HomeController.php]: Should it be an end comment tag on row 98?

Comment (wakerman17) [reply cloudserver/app/Http/Controllers/HomeController.php]: Removed

**Review by MexxII — APPROVED** [review]
Looks pretty good!

File: cloudserver/app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -27,9 +27,56 @@ public function index()
...
+			return 	view('/home')
+					->with('user_devices', $user_devices)
+					->with('ip', $rasp[0]->ip_address)
+					->with('flag', $flag);
+		} 
+		if  (count($rasp) === 0) 
+		{
+			$flag = 2;
+			return 	view('/home')
+					->with('flag', $flag);
```
Comment (anga13) [inline cloudserver/app/Http/Controllers/HomeController.php]: Kommentar om vad flag-variabeln innebär eller tydligare variabelnamn vore bra

**Comment by anga13** [comment]
Great!

**Review by anga13 — APPROVED** [review]

**Pull request merged**